README.md

```diff
@@ -84,26 +84,42 @@ Please note — packaged into an asar archive [by default](https://github.com/el
 * `-amd64.deb` and `-i386.deb`: Linux Debian package. Please note — by default the most effective [xz](https://en.wikipedia.org/wiki/Xz) compression format used.
 
 You need to deploy somewhere releases/downloads server.
-Consider to use [Nuts](https://github.com/GitbookIO/nuts) (GitHub as a backend to store assets) or [Electron Release Server](https://github.com/ArekSredzki/electron-release-server).
+Consider using [Nuts](https://github.com/GitbookIO/nuts) (GitHub as a backend to store assets) or [Electron Release Server](https://github.com/ArekSredzki/electron-release-server).
 
 # Code Signing
 OS X and Windows code singing is supported.
-On a development machine set environment variable `CSC_NAME` to your identity (recommended). Or pass `--sign` parameter.
+
+## Signing OS X Development Builds
```

Comment:
No one forbid you to build production build on a development machine.

The only difference here — is that you can use your local keychain / pass `--sign` param explicitly. I suggest to leave this sentence as is (not to extract it to separate section). But add note that `CSC_NAME` works only on OS X.